astroplan/core.py

```diff
@@ -1128,8 +1187,44 @@ def moon_position(self, time):
             the `~astropy.time.Time` initializer, so it can be anything that
             `~astropy.time.Time` will accept (including a `~astropy.time.Time`
             object).
+
+        Returns
+        -------
+        altaz : `~astropy.coordinates.SkyCoord`
+            Position of the moon transformed to altitude and azimuth
         """
-        raise NotImplementedError()
+        if not isinstance(time, Time):
+            time = Time(time)
+
+        try:
+            import ephem
+        except ImportError:
+            raise ImportError("The moon_altaz function currently requires "
+                              "PyEphem to compute the position of the moon.")
+
+        moon = ephem.Moon()
+        obs = ephem.Observer()
+        obs.lat = self.location.latitude.to(u.degree).to_string(sep=':')
+        obs.lon = self.location.longitude.to(u.degree).to_string(sep=':')
+        obs.elevation = self.location.height.to(u.m).value
+        if self.pressure is not None:
+            obs.pressure = self.pressure.to(u.bar).value*1000.0
+
+        if time.isscalar:
+            obs.date = time.datetime
+            moon.compute(obs)
+            moon_alt = float(moon.alt)
+            moon_az = float(moon.az)
+        else:
+            moon_alt = []
+            moon_az = []
+            for t in time:
+                obs.date = t.datetime
+                moon.compute(obs)
+                moon_alt.append(float(moon.alt))
+                moon_az.append(float(moon.az))
+        return SkyCoord(alt=moon_alt*u.rad, az=moon_az*u.rad, frame='altaz',
+                        obstime=time)
```

Comment:
You'll need to pass in all the other elements of the altaz frame that apply for this observer.  The easiest way is probably:
```python
SkyCoord(alt=moon_alt*u.rad, az=moon_az*u.rad, frame=self.altaz(time))
```